server/routes_test.go

```diff
@@ -106,11 +106,11 @@ func TestServerRoutesWithAuthAndBCrypt(t *testing.T) {
 	srvB := RunServer(optsB)
 	defer srvB.Shutdown()
 
-	urlA := fmt.Sprintf("nats://%s:%d/", optsA.Host, optsA.Port)
-	urlB := fmt.Sprintf("nats://%s:%d/", optsB.Host, optsB.Port)
+	urlA := fmt.Sprintf("nats://%s:%s@%s:%d/", optsA.Username, optsA.Password, optsA.Host, optsA.Port)
+	urlB := fmt.Sprintf("nats://%s:%s@%s:%d/", optsB.Username, optsB.Password, optsB.Host, optsB.Port)
 
 	// Wait for route to form.
-	time.Sleep(250 * time.Millisecond)
+	time.Sleep(4 * time.Second)
```

Comment:
Does not have to be part of this PR. The thing is since this is an util function that may be called in several places in the same test function, it is better to use `stackFatalf` to know exactly where in the test function it failed.